distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/util/RendezvousHash.java

```diff
@@ -0,0 +1,104 @@
+package ru.fix.distributed.job.manager.util;
```

Comment:
https://github.com/clohfink/RendezvousHash

Reply:
The idea was to provide link to gihub project in our source code.   
No one  going to look into Pull request history.   
Suggestion: add first line comment to this file, that you get it from gihub project and provide this link here.